src/main/java/org/apache/commons/lang3/ObjectUtils.java

```diff
@@ -951,6 +951,25 @@ their constant using one of the CONST() utility methods, instead:
             public final static int MAGIC_NUMBER = CONST(5);
      */
 
+    /**
+     * <p>Determine whether the given object is an array:
```

Comment:
I've been using "Tests..." for `is*` methods. If you really want to keep "Determine" then it should be "Determines", also the main part of the Javadoc should mention that the method is null-safe, for example: "Test whether the given object is an Object array or a primitive array in a null-safe manner.".

Reply:
Changed.
TY